requirements.txt

```diff
@@ -1,32 +1,10 @@
-antlr4-python3-runtime==4.7
-certifi==2017.7.27.1
-chardet==3.0.4
-codeclimate-test-reporter==0.2.3
-contextvars==2.2
-coverage==4.4.1
-cycler==0.10.0
-flake8==3.4.1
-idna==2.6
-matplotlib==2.0.2
-mccabe==0.6.1
-mock==2.0.0
-numpy==1.13.3
-pbr==3.1.1
-pluggy==0.5.2
-py==1.4.34
-pycodestyle==2.3.1
-pyflakes==1.5.0
-pyparsing==2.2.0
-pytest==3.2.3
-pytest-cov==2.5.1
-python-dateutil==2.6.1
-pytz==2017.2
-requests==2.18.4
-requests-mock==1.3.0
-six==1.11.0
-Sphinx==1.6.6
-sphinx-rtd-theme==0.2.4
-tox==2.9.1
-typing==3.6.2
-urllib3==1.22
-virtualenv==15.1.0
+# runtime deps
+numpy
+antlr4-python3-runtime
+requests
+contextvars
+six
```

Comment:
why do we still need six?

Reply:
we haven't removed it yet. I started going through and removing the places where we use `six` but I got bored